tools/release-plugin.sh

```diff
@@ -36,6 +36,11 @@ if ! command -v gh &> /dev/null; then
 	fi 
 fi
 
+GH_VER="$( gh --version | grep -E -o -m1 '([0-9])*\.([0-9])*\.([0-9])*' )"
```

Comment:
The parens do nothing here. OTOH, `+` instead of `*` could be useful to avoid accidentally matching ".." should that appear in the output.
```suggestion
GH_VER="$( gh --version | grep -E -o -m1 '[0-9]+\.[0-9]+\.[0-9]+' )"
```